README.md

```diff
@@ -48,23 +48,23 @@ wifi.sta.config{ssid="SSID", pwd="password"}
 
 The entire [NodeMCU documentation](https://nodemcu.readthedocs.io) is maintained right in this repository at [/docs](docs). The fact that the API documentation is maintained in the same repository as the code that *provides* the API ensures consistency between the two. With every commit the documentation is rebuilt by Read the Docs and thus transformed from terse Markdown into a nicely browsable HTML site at [https://nodemcu.readthedocs.io](https://nodemcu.readthedocs.io).
 
-- How to [build the firmware](https://nodemcu.readthedocs.io/en/master/en/build/)
-- How to [flash the firmware](https://nodemcu.readthedocs.io/en/master/en/flash/)
-- How to [upload code and NodeMCU IDEs](https://nodemcu.readthedocs.io/en/master/en/upload/)
+- How to [build the firmware](https://nodemcu.readthedocs.io/en/release/build/)
+- How to [flash the firmware](https://nodemcu.readthedocs.io/en/release/flash/)
+- How to [upload code and NodeMCU IDEs](https://nodemcu.readthedocs.io/en/release/upload/)
```

Comment:
Thanks. It's good we fix these even though the necessary redirects have long been in place.

Reply:
Nope. Maybe the redirect to release is done first.